src/parser/retail/modules/items/EnchantChecker.tsx

```diff
@@ -89,7 +78,6 @@ const MIN_ENCHANT_IDS = [
   ITEMS.ENCHANT_WEAPON_WAFTING_DEVOTION_R2,
   ITEMS.ENCHANT_WEAPON_BURNING_WRIT_R1,
   ITEMS.ENCHANT_WEAPON_BURNING_WRIT_R2,
-  ITEMS.ENCHANT_WEAPON_BURNING_WRIT_R3,
```

Comment:
Was this intended to be removed?

Reply:
It was not; I'll get it taken care of when I get back home.